internal/testing/e2e/consul.go

```diff
@@ -24,8 +25,10 @@ import (
 )
 
 const (
-	consulImage          = "hashicorp/consul:1.11.2"
-	configTemplateString = `
+	defaultConsulImage            = "hashicorp/consul:1.11.2"
```

Comment:
Would it be possible to pull this default up a few layers to somewhere more obvious, or maybe even remove it and require it to be set explicitly? Any thoughts on using something like https://github.com/joho/godotenv for default testing env?

Reply:
godotenv reminded me of [rakyll/globalconf](https://github.com/rakyll/globalconf) which I've used in almost every microservice I've built over the last few years. The beauty is that you can provide config values in `.ini` files, flags to your run command, or in the environment. Each value maps to a stdlib flag in your application code.